hack/cluster.sh

```diff
@@ -51,7 +52,7 @@ function up() {
     kind delete cluster --name $cluster_name
     (
         cd ${OVN_KUBERNETES_DIR}
-        ./contrib/kind.sh --local-kind-registry -ic -ikv -i6 -mne -cn ${cluster_name}
+        ./contrib/kind.sh --local-kind-registry $KIND_ARGS -cn ${cluster_name}
```

Comment:
could we for consistency use `${KIND_ARGS}` instead ?

Reply:
done